### ISSUE TITLE
cEOS - Cli: executable file not found in $PATH: unknown

### ISSUE THREAD

**Issue by Spiritus44** [srl-labs/containerlab#782]
Hello everyone,

This morning I tried to use containerlab on macos using the docker command list in the documentation :
```
docker run --rm -it --privileged \                                                
    --network host \  
    -v /var/run/docker.sock:/var/run/docker.sock \
    -v /run/netns:/run/netns \
    --pid="host" \
    -w $CLAB_WORKDIR \
    -v $CLAB_WORKDIR:$CLAB_WORKDIR \
    ghcr.io/srl-labs/clab bash
```

Once the containers instanciated I tried to log in, Cli is not found :
```
bash-5.1# docker ps
CONTAINER ID   IMAGE                   COMMAND                  CREATED             STATUS             PORTS     NAMES
d091887ee2fe   ceos:4.27.2F            "/sbin/init systemd.…"   About an hour ago   Up About an hour             clab-qinq-ce2
f7534358410d   ceos:4.27.2F            "/sbin/init systemd.…"   About an hour ago   Up About an hour             clab-qinq-pe1
85dd049380cd   ceos:4.27.2F            "/sbin/init systemd.…"   About an hour ago   Up About an hour             clab-qinq-ce1
09bceda6e43d   ceos:4.27.2F            "/sbin/init systemd.…"   About an hour ago   Up About an hour             clab-qinq-pe2
bash-5.1# docker exec -it 09bceda6e43d Cli
OCI runtime exec failed: exec failed: container_linux.go:380: starting container process caused: exec: "Cli": executable file not found in $PATH: unknown
```

Next I went to the container to show if the Cli binary exist in the path, I didn't find it:

```
bash-4.2# echo $PATH
/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
bash-4.2# for i in $(echo $PATH | tr ":" "\n"); do
>   ls $i | grep -i cli
> done
wpa_cli
wpa_cli-fips
wpa_cli-nofips
cliribd
dhclient
dhclient-script
AclCvxClient
AleFibCliDiffUtil
BgpCliHelper
CliCtrl
CliExtensionChecker.py
CliShell
ControllerClient
ControllerDebugClient
EapiCli
FastCli
McsClient
MssClient
RunCli
SessionCli
StunClient
TwinCvxClient
XmppCli
cliIsis
cliOspf
cliOspf3
cliRip
dirmngr-client
fastclient
ivshmem-client
ovsdb-client
rCli
redis-cli
turnutils_stunclient
turnutils_uclient
zerotouch-dhclient-script
cliribd
dhclient
dhclient-script
AclCvxClient
AleFibCliDiffUtil
BgpCliHelper
CliCtrl
CliExtensionChecker.py
CliShell
ControllerClient
ControllerDebugClient
EapiCli
FastCli
McsClient
MssClient
RunCli
SessionCli
StunClient
TwinCvxClient
XmppCli
cliIsis
cliOspf
cliOspf3
cliRip
dirmngr-client
fastclient
ivshmem-client
ovsdb-client
rCli
redis-cli
turnutils_stunclient
turnutils_uclient
zerotouch-dhclient-script
```

Do you know what I'm doing wrong?

**Comment by hellt** [comment]
Hi @Spiritus44 
I personally didn't try to launch cEOS on Mac OS... 

What I'd suggest is to launch it on a linux vm and check where `Cli` binary is. Alternatively you may ask this question on containerlab discord in arista channel

**Comment by hellt** [comment]
Hi @Spiritus44 
I saw this was solved in discord with a suggestion to try 4.28? It seems the root cause was with cgroups

Are we good closing this?

**Comment by Spiritus44** [comment]
Tank you @hellt for the remind.
Yes, I close this ticket, it solved.

To anyone who encounter the same issue, all you have to do is to use the 4.28> cEOS version that you can find on the Arista cEOS offical website under the EFT section.

This issue was added by upgrading docker desktop which use by default cgroup2.

Thank a lot to the community and in particular for the help of noredistribution on the discord channel.
